packages/common/src/groups/HubGroups.ts

```diff
@@ -208,12 +210,14 @@ export async function updateHubGroup(
   await updateGroup(opts);
 
   // create or update entity settings
-  const entitySetting = await createOrUpdateEntitySettings(
-    hubGroup,
-    context.hubRequestOptions
-  );
-  hubGroup.entitySettingsId = entitySetting.id;
-  hubGroup.discussionSettings = entitySetting.settings.discussions;
+  if (!context.isPortal) {
+    const entitySetting = await createOrUpdateEntitySettings(
+      hubGroup,
+      context.hubRequestOptions
+    );
+    hubGroup.entitySettingsId = entitySetting.id;
+    hubGroup.discussionSettings = entitySetting.settings.discussions;
+  }
 
   return hubGroup;
```

Comment:
We also need to make sure no fetches, creates, or updates are called for discussion settings in the context of sites.